llvm/lib/Target/X86/X86InstrCompiler.td

```diff
@@ -726,6 +746,18 @@ let usesCustomInserter = 1,
     defm CTSELECT_VR64   : CTSELECT_NOCMOV<VR64, x86mmx>;
```

Comment:
Could you add support for other `CTSELECT` variant (`CTSELECT_FP32`, `CTSELECT_FP64`, `CTSELECT_FP80` and `CTSELECT_VR64`) using CTSELECT_I386 class?